dev/io.openliberty.reporting.internal_fat/publish/servers/io.openliberty.reporting.response.app.server/server.xml

```diff
@@ -38,6 +38,6 @@
     <!-- <webApplication contextRoot="/" /> -->
 
     <!-- Default SSL configuration enables trust for default certificates from the Java runtime -->
-    <ssl id="defaultSSLConfig" trustDefaultCerts="true" />
+	<ssl id="defaultSSLConfig" trustDefaultCerts="true" />
```

Comment:
`trustDefaultCerts` needs to be removed as it adds the jvm store into the overall store used by TS for its truststore, so we don't know if we are using just the JVM or the trustStore defined here

Reply:
Done